rllib/models/tests/test_base_model.py

```diff
@@ -0,0 +1,189 @@
+import unittest
+
+from ray.rllib.models.base_model import (
+    UnrollOutputType,
+    Model,
+    RecurrentModel,
+    ForwardOutputType,
+)
+import numpy as np
+from ray.rllib.models.temp_spec_classes import TensorDict, SpecDict
+from ray.rllib.utils.test_utils import check
+
+
+class NpRecurrentModelImpl(RecurrentModel):
+    """A numpy recurrent model for checking:
+    (1) initial states
+    (2) that model in/out is as expected
+    (3) unroll logic
+    (4) spec checking"""
+
+    input_spec = SpecDict({"in": "h"}, h=3)
```

Comment:
override the properties instead of doing it like this.

Reply:
Fixed